README.md

```diff
@@ -1 +1,93 @@
-# sawtooth-sdk-ios
+
+![Hyperledger Sawtooth](https://raw.githubusercontent.com/hyperledger/sawtooth-core/master/images/sawtooth_logo_light_blue-small.png)
+
+# Hyperledger Sawtooth SDK
+
+*Hyperledger Sawtooth* is an enterprise solution for building, deploying, and
+running distributed ledgers (also called blockchains). It provides an extremely
+modular and flexible platform for implementing transaction-based updates to
+shared state between untrusted parties coordinated by consensus algorithms.
+
+The *Sawtooth SDK* provides a number of useful components that simplify
+developing Swift applications which interface with the Sawtooth platform.
+
+## Installation
+
+The Swift SDK's SawtoothSigning module can be imported to an iOS project via
+[Carthage](https://github.com/Carthage/Carthage), an dependency manager for
+iOS.
```

Comment:
Carthage can be used with any of the Cocoa APIs

Reply:
also slight typo "an" -> "a"

Reply:
Updated